### ISSUE TITLE
error creating CMS Fortress Users

### ISSUE THREAD

**Issue by ferrisoxide** [melvinsembrano/cms-fortress#80]
The migration `devise_create_cms_fortress_users.cms_fortress_engine.rb` fails under Rails 4.2.1 using SQLite, with the following error:

```
ActiveRecord::StatementInvalid: SQLite3::SQLException: no such column: cms_fortress_users.site_id: SELECT  1 AS one FROM "cms_fortress_users" WHERE ("cms_fortress_users"."email" = 'admin@cmsfortress.com' AND "cms_fortress_users"."site_id" IS NULL) LIMIT 1
[path to migration file]_devise_create_cms_fortress_users.cms_fortress_engine.rb:8:in `migrate'
SQLite3::SQLException: no such column: cms_fortress_users.site_id
[path to migration file]_devise_create_cms_fortress_users.cms_fortress_engine.rb:8:in `migrate'
```

It appears that the `migration` method is running before the `change`. The work around is to comment out the ` Cms::Fortress::User.create!` code, run the migration and then execute the offending code in the console.

Hope that's useful. Great gem btw (from experience using this in the past). Looking forward to seeing what's new.

**Comment by melvinsembrano** [comment]
Issue replicated, thanks @ferrisoxide for reporting.

**Comment by melvinsembrano** [comment]
This is now fixed on version 1.3.9
